File: api/code-samples/python/eth/gas/feeHistory.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockCount = "0x10"
lastBlock = "latest"
rewardPercentiles = [0.1, 0.2, 0.3]

sdk = OpenSDK(host)
eth_response = sdk.eth.fee_history(blockCount, lastBlock, rewardPercentiles)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/eth/gas/feeHistory.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockCount = '0x10'
    const lastBlock = 'latest'
    const rewardPercentiles = [0.1, 0.2, 0.3]
    
    sdk.eth.feeHistory(blockCount, lastBlock, rewardPercentiles, {}, (err, data, response) => {
        console.log(data);
    });
}
)()